File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function (nums) {
    let evenCount = 0;

    nums.forEach(ele => {
        if ((ele >= 10 && ele <= 99) || (ele >= 1000 && ele <= 9999) || ele >= 100000) {
            evenCount++;
        }
    })

    return evenCount;
};

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int count = 0;

        for(int i = 0; i < nums.size(); i++) {
            int ele = nums[i];
            int digitCount = 0;

            while(ele) {
                ele = ele/10;
                digitCount++;
            }

            if(digitCount % 2 == 0) {
                count++;
            }
        }
        
        return count;
    }

};